src/Lookup/UnresolvedEntityRedirectException.php

```diff
@@ -22,10 +21,11 @@ class UnresolvedRedirectException extends RuntimeException {
 	private $redirectTargetId;
 
 	/**
+	 * @param EntityId $entityId
 	 * @param EntityId $redirectTargetId The ID of the target Entity of the redirect
 	 */
-	public function __construct( EntityId $redirectTargetId ) {
-		parent::__construct( "Unresolved redirect to " . $redirectTargetId->getSerialization() );
+	public function __construct( EntityId $entityId, EntityId $redirectTargetId ) {
+		parent::__construct( $entityId, "Unresolved redirect to " . $redirectTargetId->getSerialization() );
```

Comment:
Wikibase still needs the revision ID and Timestamp here. Or shall we use a subclass?